src/index.tsx

```diff
@@ -358,6 +366,11 @@ export default class BottomSheetBehavior extends React.Component<Props, State> {
     // current snap point desired
     this.snapPoint = currentSnapPoint()
 
+    if (props.enabledBottomClamp) {
+      this.clampingEnabled.setValue(1);
```

Comment:
You can move it directly to declaration :)

Reply:
@osdnk I'm not totally sure if you meant animation declaration or class declaration but hopefully I got you right :)